home-data-migration/src/main/resources/db/changelog/0.0.1/Issue304.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.3.xsd">
+
+    <changeSet id="Issue304.001" author="klaeck">
+        <dropColumn tableName="invitations" columnName="cooperation_name"/>
+        <addColumn tableName="invitations">
+            <column name="cooperation_id" type="bigint">
+                <constraints nullable="true" foreignKeyName="cooperation_fk" references="cooperations(id)"/>
+            </column>
+        </addColumn>
+    </changeSet>
+
+    <changeSet id="Issue304.002" author="klaeck">
+        <dropUniqueConstraint tableName="cooperations" constraintName="cooperation_usero_key"/>
+<!--        <dropUniqueConstraint tableName="cooperations" constraintName="cooperation_iban_key"/>-->
+        <dropNotNullConstraint tableName="cooperations" columnName="name"/>
+        <dropNotNullConstraint tableName="cooperations" columnName="iban"/>
+        <dropNotNullConstraint tableName="cooperations" columnName="address_id"/>
+    </changeSet>
+
+    <changeSet id="Issue304.003" author="klaeck">
+        <addColumn tableName="cooperations">
+            <column name="admin_email" type="VARCHAR(50)">
+                <constraints nullable="false"/>
+            </column>
+        </addColumn>
+    </changeSet>
+
+    <changeSet id="Issue304.004" author="klaeck">
+        <addUniqueConstraint tableName="cooperations"
+                             columnNames="usreo, admin_email"
+        constraintName="existing_constraint"/>
+    </changeSet>
+</databaseChangeLog>
```

Comment:
Please add newline at the end of the file. see https://stackoverflow.com/a/5813359